docs/locale/ja/LC_MESSAGES/annotations.po

```diff
@@ -0,0 +1,246 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-2018, spotbugs community
+# This file is distributed under the same license as the spotbugs package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
```

Comment:
Should probably populate this.

Reply:
It seems that the email address is not configurable. Let's put it aside for now.
https://shimizukawa-sphinx.readthedocs.io/en/latest/_modules/sphinx/builders/gettext.html